### PR TITLE
Catalog and SP generic updates

### PR DESCRIPTION
**Pull request by quinntracy**

### CODE REVIEW

**Comment by quinntracy** [comment]
Some screenshots need to be updated to match the current UI. Should this be done with 10.5?

File: content/en/docs/catalog/integrate/share-data/_index.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Do the following:
 
 4. Name the published service *{yourname}CustomerODataService* and click **OK**.
 
-    The new **{yourname}CustomerODataService** is added to the module and the **Edit published resource** dialog box is displayed for the entity **Customer**. 
+    The new **{yourname}CustomerODataService** is added to the module. Select which attribute(s) to use as the key of this entity, then click **OK**. The **Edit published resource** dialog box is displayed for the entity **Customer**.
```
Comment (quinntracy) [inline content/en/docs/catalog/integrate/share-data/_index.md:86]: What information should be shared to explain the purpose of the **Capabilities** section in the **Edit published resource** dialog box?

File: content/en/docs/catalog/integrate/share-data/_index.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Do the following:
 
 1. In the App Explorer, right-click **MyFirstModule**. From the drop-down list, select **Add a folder**. Name it  *APIs*.
 
-    {{% alert color="info" %}}The published OData service functions as an API to your app. Some apps may have several published services, so it is good practice to keep them together in a folder for each module.{{% /alert %}}
+    {{% alert color="info" %}}The published service functions as an API to your app. Some apps may have several published services, so it is good practice to keep them together in a folder for each module.{{% /alert %}}
 
 2. In the **Domain Model**, right-click the **Customer** entity and select **Expose as OData resource…**.
```
Comment (quinntracy) [inline content/en/docs/catalog/integrate/share-data/_index.md:75]: Should we add steps to **Publish in REST service**?

File: content/en/docs/catalog/integrate/share-data/_index.md [hunk 3]
```diff
@@ -70,33 +70,33 @@ Do the following:
...
 
     {{< figure src="/attachments/catalog/share-data/expose-as-odata-resource.png" >}}
 
-3. In the **Select Published OData Service** dialog box, select the **MyFirstModule** > **APIs** folder and click **New** to add a new OData service to this folder.
+3. In the **Select Published OData Service** dialog box, select the **MyFirstModule** > **APIs** folder and click **New** to add a new service to this folder.
```
Comment (NammNguyen) [inline content/en/docs/catalog/integrate/share-data/_index.md:79]: This document is OData specific though, does it need to be generalized?

Comment (NammNguyen) [reply content/en/docs/catalog/integrate/share-data/_index.md:79]: This applies for all changes made to this document

File: content/en/docs/catalog/integrate/share-data/_index.md (hunk 1 above)
Comment (NammNguyen) [reply content/en/docs/catalog/integrate/share-data/_index.md:86]: Do we not have documentation elsewhere regarding capabilities?

File: content/en/docs/catalog/integrate/write-data/_index.md [hunk 4]
```diff
@@ -10,13 +10,13 @@ aliases:
 
 ## 1 Introduction
 
-This how-to builds on the information provided in [How to Share Data Between Apps](/catalog/write-data/) and shows you how OData annotations give you the power to create, update, and delete data.
+This how-to builds on the information provided in [How to Share Data Between Apps](/catalog/write-data/) and shows you how annotations give you the power to create, update, and delete data.
```
Comment (NammNguyen) [inline content/en/docs/catalog/integrate/write-data/_index.md:13]: Writing data is also OData specific I think

Comment (NammNguyen) [reply content/en/docs/catalog/integrate/write-data/_index.md:13]: Also for the other changes in this doc

**Pull request closed without merging**